app/views/users/activate_account.html.erb

```diff
@@ -10,14 +10,25 @@
 </div>
 
 <%- content_for(:no_ember_head) do %>
+  <%= script "preload_store" %>
```

Comment:
Sure. I was trying to make this similar to all the other places where `.ajax` is called (which either use `Discourse.ajax` or `$.ajax(Discourse.getURL('/users/search/users')...` What you are suggesting would make for simpler/smaller code, but different from any other ajax call. Not sure which is better.

Not sure